include/libopencm3/stm32/common/dma_common_l1f013.h

```diff
@@ -422,4 +424,3 @@ END_DECLS
 /** @endcond */
 
 /**@}*/
-
```

Comment:
Sorry, my editor does that. I usually remove those changes again when submitting something but I though it doesn't really hurt at the end of the file since even a git blame wouldn't really show that anyways.